app/v2/controllers/requestParsers/validators/validations/ConsolidatedExpensesValidation.scala

```diff
@@ -16,11 +16,12 @@
 
 package v2.controllers.requestParsers.validators.validations
 
-import v2.models.errors.{MtdError, RuleBothExpensesSuppliedError}
+import v2.models.errors.{ MtdError, RuleBothExpensesSuppliedError }
 import v2.models.request.common.foreignFhlEea._
 import v2.models.request.common.foreignPropertyEntry._
-import v2.models.request.common.ukFhlProperty.UkFhlPropertyExpenses
-import v2.models.request.common.ukNonFhlProperty.UkNonFhlPropertyExpenses
+import v2.models.request.common.ukFhlProperty.{ UkFhlPropertyExpenses => CommonUkFhlPropertyExpenses }
```

Comment:
Not sure if these really need to be renamed to "Common..."?

Reply:
There are two classes with the name UkFhlPropertyExpenses - the one in common and the one in createHistoricNonFhlUkPropertyPeriodSummary - so to avoid conflict one needed to be renamed